syncer/syncer.go

```diff
@@ -89,9 +89,13 @@ type Syncer struct {
 	lastATXsSynced    atomic.Value
 
 	mu sync.Mutex
+
 	// awaitSyncedCh is the list of subscribers' channels to notify when this node enters synced state
 	awaitSyncedCh []chan struct{}
```

Comment:
this can be removed. only activation pkg uses this, and you can replace them with the new api.